consensus/propagation/Reactor.go

```diff
@@ -18,19 +22,32 @@ const (
 
 	// ReactorIncomingMessageQueueSize the size of the reactor's message queue.
 	ReactorIncomingMessageQueueSize = 1000
+
+	// Channel the channel ID used by the propagation reactor.
+	Channel = byte(0x50)
+
+	// DataChannel Duplicate of consensus data channel
+	// added as a temporary fix for circular dependencies
+	// TODO: fix redefining these values here.
+	DataChannel  = byte(0x21)
+	StateChannel = byte(0x20)
```

Comment:
we need at least two channels 1) recovery parts, compact blocks / proposals, and haves must be in the same channel 2) its likely advantageous to have another channel for wants